src/commands/chatters/artemis.ts

```diff
@@ -3,51 +3,60 @@ import { Discord, SimpleCommand, SimpleCommandMessage, SimpleCommandOption, Simp
+    'Tolkien did not receive the total amount he was owed for the LotR Trilogy.',
+    'Tolkien originally did not want to name the third book The Return of the King because it revealed too much about the story.',
+    'Tolkien and his wife\'s gravestones are engraved with two names from the Silmarillion, Beren and Luthien, the characters in "the greatest love story he ever wrote" according to Tolkien.',
+    'The University of California offers a LotR dorm for its freshmen.',
+    "We don't talk about the Hobbit films.",
+    'Frodo is depicted as a clumsy character in the films, falling 39 times.',
+    "Christopher Lee is the only cast member to have met Tolkien, and always wanted to play the role of Gandalf which he received Tolkien's blessing to play.",
+    'Nicolas Cage was originally supposed to play the role of Aragorn.',
+    'The fight scenes were choreographed by an Olympic fencer, who described Viggo Mortensen as "the best Sword fighter i have ever trained"',
+    'Viggo Mortensen did his stunts without a double (he also broke his toe…)',
+    'Orlando Bloom broke his ribs during filming.',
+    'John Rhy-Davies (Gimli) is the tallest actor in the fellowship.',
+    "Peter Jackson's daughter makes several cameos throughout the films.",
+    'Denethor could not light the pyre on fire.',
+    'The Queen of Denmark illustrated the Danish edition of LotR.',
+    'Nazgul screeches were made with plastic cups.',
+    'The game of golf canonically exists in Middle Earth.',
+    'The Gollum Effect was loosely based on the withdrawal symptoms of heroin addicts.',
+    "The battle of Helm's Deep took 4 months to film.",
+    'Did you know, the Urukhai who threw the knife at Aragorn was supposed to miss, but he got too into the scene, but then Viggo simply "Aragorned" the knife away?',
+    'Gollums were originally kind creatures.',
+    "The Dead Marshes were said to be based on Tolkien's experience in WWI.",
+    'Sean Astin was knocked unconscious during filming. He also suffered a deep laceration to his foot.',
+    'The LotR franchise has won every award it was nominated for.',
+    'Jake Gyllenhaal auditioned to play Frodo.',
+    'Vin Diesel, Liam Neeson, and Uma Thurman were up for roles.',
+    'Bill the pony was two people in a horse costume.',
+    'Sean Bean was reading his script during the council of Elrond scene.',
+    "The uruk-hai at Helm's Deep are New Zealand cricket fans.",
+    'Aragorn is the hottest character in the movies.',
+    'Most of the Rohirrim are actually women in fake beards, as the production found it hard to find enough male horse riders.',
+    "Weta Workshop had to keep asking the stuntmen playing soldiers/orcs to be more gentle with the swords they made, as they kept breaking swords and injuring each other. (The stuntmen didn't listen)",
   ]
 
   @SimpleCommand({ name: 'artemis', description: 'Artemis', prefix: '!' })
```

Comment:
no need to include a prefix tag, we default all commands to `>` this was intentional to keep our commands from colliding with brexbot should it ever arise.